deps/llhttp/include/llhttp.h

```diff
@@ -273,6 +274,10 @@ void llhttp_settings_init(llhttp_settings_t* settings);
  * In a special case of CONNECT/Upgrade request/response `HPE_PAUSED_UPGRADE`
  * is returned after fully parsing the request/response. If the user wishes to
  * continue parsing, they need to invoke `llhttp_resume_after_upgrade()`.
+ *
+ * NOTE: if this function ever returns a non-pause type error, it will continue
+ * to return the same error upon each successive call up until `llhttp_init()`
+ * call.
```

Comment:
"call" -> "is called"?

Reply:
Good catch! This could be fixed in the next release. Would you be interested in submitting a PR: https://github.com/nodejs/llhttp/blob/master/src/native/api.h#L68 ?

Reply:
@indutny I tried to submit a quick grammar fix through the GitHub UI and accidentally committed it directly (I thought it was still under your account, not the Node.js one): https://github.com/nodejs/llhttp/commit/4ed1a54d5cfe09395c2d5cedfbbda2b5e7c792c7

Happy to back it out if it's a problem.